README.md

```diff
@@ -90,18 +89,21 @@ spec:
   containers:
   - name: traefik
     image: traefik
+    args: ["--configFile=/dynamic-config/traefik.yml"]
```

Comment:
How is now `ConfigMap` with traefik.yml mounted into the pod when you removed the `config` VolumeMount?